### PR TITLE
new.abp.io page's title and descriptions updates

### PR DESCRIPTION
**Pull request by dgnyldrm7**
Related [4299](https://github.com/volosoft/vs-internal/pull/4660)

### CODE REVIEW

File: abp_io/AbpIoLocalization/AbpIoLocalization/Base/Localization/Resources/en-GB.json [hunk 1]
```diff
@@ -38,6 +38,6 @@
     "OrganizationOverview": "Organization Overview",
     "EmailPreferences": "Email Preferences",
     "DoYouAgreePrivacyPolicy": "By clicking <b>Subscribe</b> button you agree to the <a href=\"https://account.abp.io/Account/TermsConditions\">Terms & Conditions</a> and <a href=\"https://account.abp.io/Account/Privacy\">Privacy Policy</a>.",
-    "AbpConferenceDescription": "ABP Conference is a virtual event for .NET developers to learn and connect with the community."
+    "AbpConferenceDescription": "On 25, 26 September 2024, we’re hosting a free, live, virtual event where community speakers cover topics about .NET and soft-skills."
```
Comment (salihozkara) [inline abp_io/AbpIoLocalization/AbpIoLocalization/Base/Localization/Resources/en-GB.json]: This is a generic key please create a new key or take the date as a variable

File: abp_io/AbpIoLocalization/AbpIoLocalization/Base/Localization/Resources/en.json [hunk 2]
```diff
@@ -250,7 +250,7 @@
     "EmailPreferences": "Email Preferences",
     "VideoCourses": "Essential Videos",
     "DoYouAgreePrivacyPolicy": "By clicking <b>Subscribe</b> button you agree to the <a href=\"https://account.abp.io/Account/TermsConditions\">Terms & Conditions</a> and <a href=\"https://account.abp.io/Account/Privacy\">Privacy Policy</a>.",
-    "AbpConferenceDescription": "ABP Conference is a virtual event for .NET developers to learn and connect with the community.",
+    "AbpConferenceDescription": "On 25, 26 September 2024, we’re hosting a free, live, virtual event where community speakers cover topics about .NET and soft-skills.",
```
Comment (salihozkara) [inline abp_io/AbpIoLocalization/AbpIoLocalization/Base/Localization/Resources/en.json]: This is a generic key please create a new key or take the date as a variable

File: abp_io/AbpIoLocalization/AbpIoLocalization/Www/Localization/Resources/en.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "culture": "en",
   "texts": {
-    "GetStarted": "Get Started",
+    "GetStarted": "Get Started with ABP | Quick and Easy Setup Guide",
```
Comment (salihozkara) [inline abp_io/AbpIoLocalization/AbpIoLocalization/Www/Localization/Resources/en.json]: This is a generic key please create a new key

**Review by salihozkara — CHANGES_REQUESTED** [review]
Make sure that the key whose contents you are changing is not used elsewhere. Or create a new key

**Review by salihozkara — APPROVED** [review]

**Review by oykuermann — CHANGES_REQUESTED** [review]
Waiting for https://github.com/volosoft/vs-internal/pull/4660#issuecomment-2348666650

**Review by salihozkara — APPROVED** [review]

**Review by oykuermann — APPROVED** [review]

**Pull request merged**